### PR TITLE
✨ feat: oracle failsafe

### PR DESCRIPTION
**Pull request by j-vp**

### CODE REVIEW

**Comment by Thomas-Smets** [comment]
Shouldn't we remove all setOracles() from PricingModules?

File: src/test/1_OracleHub.t.sol [hunk 1]
```diff
@@ -1081,4 +1124,243 @@ contract OracleHubTest is Test {
...
+                isActive: true
+            })
+        );
+        vm.stopPrank();
+
+        vm.warp(2 weeks); //to not run into an underflow
+
+        vm.startPrank(oracleOwner);
+        //minAnswer is set to 100 in the oracle mocks
+        oracleSnxToEth.transmit(int256(1));
```
Comment (Thomas-Smets) [inline src/test/1_OracleHub.t.sol:1263]: Don't also set price to low

File: src/test/1_OracleHub.t.sol [hunk 2]
```diff
@@ -1081,4 +1124,243 @@ contract OracleHubTest is Test {
...
+        oracleEthToUsd.transmit(int256(500000000000));
```
Comment (Thomas-Smets) [inline src/test/1_OracleHub.t.sol:1207]: What's the point of this oracle? Should we also try to decommission it?

File: src/test/1_OracleHub.t.sol [hunk 3]
```diff
@@ -1081,4 +1124,243 @@ contract OracleHubTest is Test {
...
+            })
+        );
+        vm.stopPrank();
+
+        vm.warp(2 weeks); //to not run into an underflow
+
+        vm.startPrank(oracleOwner);
+        //minAnswer is set to 100 in the oracle mocks
+        oracleSnxToEth.transmit(int256(1));
+        oracleEthToUsd.transmit(int256(500000000000)); //only one of the two is needed to fail
```
Comment (Thomas-Smets) [inline src/test/1_OracleHub.t.sol:1264]: What's the point of this oracle? Should we also try to decommission it?

File: src/test/1_OracleHub.t.sol (hunk 2 above)
Comment (j-vp) [reply src/test/1_OracleHub.t.sol:1207]: just checking whether it works if only one of the oracles in the train is decomissioned

**Comment by j-vp** [comment]
setOracles(), syncOracles and things related to "assetsUpdateable' are removed

**Review by Thomas-Smets — APPROVED** [review]
Gtg

**Pull request merged**